docs/advanced/debugging.md

```diff
@@ -5,122 +5,169 @@ order: 6
 
 # Debugging
 
-We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql` and those creating their own exchanges.
+We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql`
+and those creating their own exchanges.
 
-## Debug events
-
-Debug events are a means for seeing what's going on inside of `urql` exchanges. Before getting started, here are a few things to be aware of.
-
-### Anyone with client access can listen
-
-This includes the creator of an `urql` client and any of it's exchanges (although the latter is not advised).
-
-### Debug events are disabled in production
-
-Debug events are a fire-and-forget mechanism and should not be used as a means for messaging in your app.
+## Browser Devtools
 
-## Consuming debug events
+The quickest way to debug `urql` is to use the [`urql` devtools Chrome and Firefox
+extension.](https://github.com/FormidableLabs/urql-devtools/).
 
-Debugging events can be inspected either graphically using our [devtools](https://github.com/FormidableLabs/urql-devtools) or manually by subscribing to events on the client.
+It displays consists of three tools, the "Explorer", the "Events" tab, and the "Request" tab, which
+allow you to:
 
-### Devtools
-
-The quickest way is going to be using our [devtools extension](https://github.com/FormidableLabs/urql-devtools/) which visualizes events on a timeline and provides tools to filter events, inspect the cache, and trigger custom queries via the running client.
+- visualize events on a timeline as they happen
```

Comment:
We've discussed this on slack but for transparency, while I think this is great, I've no idea how we would keep all the different sources up to date with changes.

If we just link to the repo, users will be presented with up-to-date info on:
 - Features
 - Usage instructions
 - Supported platforms

This info is also documented in the store pages for chrome and firefox

Reply:
I believe we've now crunched down the "features" section to basically just answer the "What it does" but not "How it does it".

The supported platforms should be gone from your suggestions, I think.

And the usage instructions are as of now minimal and limited to installation.